wdl/CleanVcf5.wdl

```diff
@@ -1,1817 +1,302 @@
-    preemptible: select_first([runtime_override.preemptible_tries, runtime_default.preemptible_tries])
-    maxRetries: select_first([runtime_override.max_retries, runtime_default.max_retries])
-    docker: sv_pipeline_docker
-    bootDiskSizeGb: select_first([runtime_override.boot_disk_gb, runtime_default.boot_disk_gb])
-  }
+    call CleanVcf5FindRedundantMultiallelics {
+        input:
+            multiallelic_vcf=GatherMultiallelicShards.concat_vcf,
+            find_redundant_sites_script=find_redundant_sites_script,
+            sv_pipeline_docker=sv_pipeline_docker,
+            runtime_attr_override=runtime_attr_override
+    }
 
-  command <<<
-    set -euxo pipefail
-    zcat ~{cleantagandmulti_vcf}|awk '{if (($1~"X" || $1~"Y") && $1!~"#") print}'|wc -l > out
-  >>>
+    call CleanVcf5Polish {
+        input:
+            clean_gq_vcfs=CleanVcf5MakeCleanGQ.clean_gq_vcf,
+            no_sample_lists=CleanVcf5MakeCleanGQ.no_sample_list,
+            redundant_multiallelics_list=CleanVcf5FindRedundantMultiallelics.redundant_multiallelics_list,
+            sv_pipeline_docker=sv_pipeline_docker,
+            runtime_attr_override=runtime_attr_override
+    }
 
-  output {
-    Int out=read_int("out")
-  }
+    output {
+        File polished=CleanVcf5Polish.polished
+    }
 }
 
-task CleanVcf5_29_TRUE_1 {
-  input {
-    File famfile
-    File cleantagandmulti_vcf
-    String sv_pipeline_docker
-    RuntimeAttr? runtime_attr_override
-  }
-
-  Float input_size = size([famfile, cleantagandmulti_vcf], "GB")
-  RuntimeAttr runtime_default = object {
-                                  mem_gb: 7.5,
-                                  disk_gb: ceil(10.0 + input_size * 5.0),
-                                  cpu_cores: 1,
-                                  preemptible_tries: 3,
-                                  max_retries: 1,
-                                  boot_disk_gb: 10
-                                }
-  RuntimeAttr runtime_override = select_first([runtime_attr_override, runtime_default])
-  runtime {
-    memory: "~{select_first([runtime_override.mem_gb, runtime_default.mem_gb])} GB"
-    disks: "local-disk ~{select_first([runtime_override.disk_gb, runtime_default.disk_gb])} HDD"
-    cpu: select_first([runtime_override.cpu_cores, runtime_default.cpu_cores])
-    preemptible: select_first([runtime_override.preemptible_tries, runtime_default.preemptible_tries])
-    maxRetries: select_first([runtime_override.max_retries, runtime_default.max_retries])
-    docker: sv_pipeline_docker
-    bootDiskSizeGb: select_first([runtime_override.boot_disk_gb, runtime_default.boot_disk_gb])
-  }
-
-  command <<<
-    set -euxo pipefail
-    ##Determine columns male columns##
-    zcat ~{cleantagandmulti_vcf}\
-    |egrep ^# \
-    |tail -n 1 \
-    |tr '\t' '\n' \
-    |cat -n - \
-    >col.txt
+task ScatterVcf {
+    input {
+        File vcf
+        File? shard_script
+        String prefix = "shard"
```

Comment:
I'd make this required and set it to something like `~{cohort_name}.~{contig}.shard_clean_vcf_5`